<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>center2</class>
 <widget class="QWidget" name="center2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>114</width>
    <height>54</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>114</width>
    <height>54</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>114</width>
    <height>54</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>行動是治癒恐懼的良藥，而猶豫、拖延將不斷滋養恐懼</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#center2 {
background-color: rgba(255, 255, 255, 0);
}
QWidget:hover#center2{
border-image: url(:/icons/guanjia_shadow_hoverd.png);
}
QWidget#center2 {
border-image: url(:/icons/guanjia_shadow_normal.png);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #595c6b;
border-radius: 25px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>44</width>
     <height>44</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>44</width>
     <height>44</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0 red, stop:0.5 yellow stop:1 #00ffff);
border-radius: 21px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="memColorFrame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="memUse">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #595c6b;
border-radius: 19px;
font: 16pt &quot;华文细黑&quot;;
color: rgb(223, 223, 223);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>8</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>38</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>3</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #343740;
border-radius: 18px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>21</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 700 7pt &quot;Microsoft YaHei UI&quot;;
color: #549eca;</string>
        </property>
        <property name="text">
         <string>↑</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Microsoft YaHei UI&quot;;
color: #2fb634;</string>
        </property>
        <property name="text">
         <string>↓</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="netUp">
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Microsoft YaHei UI&quot;;
color: rgb(223, 223, 223);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="netDown">
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Microsoft YaHei UI&quot;;
color: rgb(223, 223, 223);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 6pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <zorder>frame_4</zorder>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>memColorFrame</zorder>
  <zorder>memUse</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
